query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/type/CswSourceConfigurationField.java

```diff
@@ -27,9 +32,21 @@
     public static final String DESCRIPTION =
             "Represents a CSW configuration containing properties to be saved.";
```

Comment:
I don't like having this description describing the other field's functions. I have made the other descriptions more meaningful.